### PR TITLE
feat(metro-config): Enable isCSSEnabled by default

### PR DESCRIPTION
**Pull request by marklawlor**
# Why

ENG-9887

# How

# Test Plan

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: templates/expo-template-tabs/metro.config.js
Comment (marklawlor) [inline templates/expo-template-tabs/metro.config.js:1]: Delete this file

File: templates/expo-template-tabs/metro.config.js [hunk 1]
```diff
@@ -2,9 +2,6 @@
 const { getDefaultConfig } = require('expo/metro-config');
 
 /** @type {import('expo/metro-config').MetroConfig} */
-const config = getDefaultConfig(__dirname, {
```
Comment (EvanBacon) [inline templates/expo-template-tabs/metro.config.js:5]: delete this file now that there are no changes.

File: docs/pages/versions/unversioned/config/metro.mdx
Comment (marklawlor) [inline docs/pages/versions/unversioned/config/metro.mdx:1]: Use tabs component

File: docs/pages/versions/unversioned/config/metro.mdx [hunk 2]
```diff
@@ -50,29 +51,40 @@ Client environment variable inlining can be disabled with the environment variab
...
+  // Disable CSS support.
+  isCSSEnabled: false,
+});
+```
 
+</Tab>
+
+<Tab label="SDK 49">
+
+Set `isCSSEnabled` to `true` in the Metro config.
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/config/metro.mdx:76]:
```suggestion
To enable, set `isCSSEnabled` to `true` in the Metro config.
```

**Review by amandeepmittal — APPROVED** [review]

File: docs/pages/versions/unversioned/config/metro.mdx [hunk 3]
```diff
@@ -50,29 +51,40 @@ Client environment variable inlining can be disabled with the environment variab
...
+<Tabs>
+
+<Tab label="SDK 50 and above">
+
+CSS support is enabled by default. You can disable the feature by setting `isCSSEnabled` in the Metro config.
 
 ```js metro.config.js
-const { getDefaultConfig } = require('expo/metro-config');
+/** @type {import('expo/metro-config').MetroConfig} */
+const config = getDefaultConfig(\_\_dirname, {
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/config/metro.mdx:66]:
```suggestion
const config = getDefaultConfig(__dirname, {
```

One small formatting issue (I think)

File: docs/pages/versions/unversioned/config/metro.mdx [hunk 4]
```diff
@@ -50,29 +51,40 @@ Client environment variable inlining can be disabled with the environment variab
...
+
+<Tab label="SDK 49">
+
+To enable CSS support, set `isCSSEnabled` to `true` in the Metro config.
+
+```js metro.config.js
 /** @type {import('expo/metro-config').MetroConfig} */
-const config = getDefaultConfig(__dirname, {
-  // Enable CSS support.
+const config = getDefaultConfig(\_\_dirname, {
```
Comment (amandeepmittal) [inline docs/pages/versions/unversioned/config/metro.mdx:80]:
```suggestion
const config = getDefaultConfig(__dirname, {
```

**Review by amandeepmittal — APPROVED** [review]

**Review by EvanBacon — APPROVED** [review]

**Pull request merged**